server/resources/translations/common.es_us.json

```diff
@@ -0,0 +1,18 @@
+{
+  "alertAcknowledged": "Otro personal de auxilio ha confirmado esta solicitud. (No necesita responder a este mensaje).",
+  "alertAdditionalAlert": "Se generó una alerta {{ alertTypeDisplayName }} adicional en {{ deviceDisplayName }}",
+  "alertCompleted": "Gracias. Esta sesión ya está completa. (No necesita responder a este mensaje).",
+  "alertFallback": "No se respondió a una alerta de verificación de {{ deviceDisplayName }}. Por favor, verifíquelo.",
+  "alertReminder": "Este es un recordatorio para que verifique {{ deviceDisplayName }}",
+  "alertStart": "Esta es una alerta {{ alertTypeDisplayName }}. Verifique en {{ deviceDisplayName }}. Responda con 'ok' una vez que lo haya verificado.",
+  "errorNoSession": "Error: No se encontró ninguna sesión activa",
+  "incidentCategorized": "El incidente se clasificó como {{ incidentCategory }}. Gracias. Esta sesión ya está completa. (No necesita responder a este mensaje).",
+  "incidentCategoryInvalid": "Lo sentimos, no se reconoció el tipo de incidente. Por favor, intente de nuevo.",
+  "incidentCategoryRequest": "Una vez que haya contestado, responda con el número que mejor describa el incidente:\n{{ incidentCategories }}",
+  "sensorDisconnectionInitial": "El Sensor Brave en {{ deviceDisplayName }} ({{ clientDisplayName }}) se ha desconectado. Presione el botón de reinicio dentro del orificio circular en la unidad de techo con un alfiler. Si no recibe un mensaje de reconexión poco después de presionar el botón de reinicio, contáctenos enviando un correo electrónico a clientsupport@brave.coop.",
```

Comment:
Good catch that we're now using `clientDisplayName` instead of `locationid` here :+1: